mayavi/modules/streamline.py

```diff
@@ -168,7 +171,8 @@ def update_pipeline(self):
 
         src = mm.source
         self.configure_connection(self.stream_tracer, src)
-        self.seed.inputs = [src]
+        if isinstance(self.seed, SourceWidget):
```

Comment:
Thanks for the PR!  I am not too happy about this `isinstance` check, it suggests that there should be a cleaner way to do this.  Is the intent to just allow a user to set the polydata and not use a widget at all?  I think it would be very useful thing in any case to allow users complete control over the points.  So maybe I should think of supporting either a widget or a poly data.  This could perhaps be done by simply having another attribute on the streamline module, say `user_seed` which is a polydata and if this is non-None, that we use that instead of the seed?